settings/options.js

```diff
@@ -1,38 +1,49 @@
 'use strict';
 
 //save user's preferences in storage
-function saveOptions(){
+function saveOptions() {
+  let overrideNewtab = document.querySelector('#newtabOverride'); 
+  browser.whimsyExp.setNewtabOverride(overrideNewtab.checked);
+  document.querySelector('#newtab-submenu').style.display = overrideNewtab.checked ? "contents" : "none";
+  
   browser.storage.sync.set({
     mario: document.querySelector('#mario').checked,
     sayings: document.querySelector('#sayings').checked,
-    newtab: document.querySelector('#newtab').value
+    newtab: document.querySelector('#newtab').value,
+    newtabOverride: overrideNewtab.checked
```

Comment:
I think we could collapse this down into a single value by using `'default'` for the non-overridden case…